src/service.rs

```diff
@@ -1394,7 +1394,13 @@ impl Service {
             "Session established with Node: {}, direction: {}",
             node_id, direction
         );
-        self.connection_updated(node_id, ConnectionStatus::Connected(enr, direction));
+        self.connection_updated(node_id, ConnectionStatus::Connected(enr.clone(), direction));
+
+        // PING immediately if the direction is outgoing. This allows us to receive a PONG without
+        // waiting for the ping_interval, making ENR updates faster.
+        if matches!(direction, ConnectionDirection::Outgoing) {
+            self.send_ping(enr, None);
+        }
```

Comment:
looking to reduce the clone I suggest moving this to the `connection_updated` fn. This way the clone is only done when the dirction is outgoing instead of every time https://github.com/ackintosh/discv5/pull/2/files

Reply:
I have merged your suggestion, thank you!